doc/sphinxext/copy_sphinxgallery.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Script to do a local install of sphinx-gallery
+rm -rf tmp sphinxgallery
+mkdir -p tmp/lib/python2.7/site-packages
+
+easy_install -Zeab tmp sphinx-gallery
+
+cp -vru tmp/sphinx-gallery/sphinxgallery/ .
```

Comment:
Before I forget you will need to add the `git add` that you added in https://github.com/nilearn/nilearn/pull/592.